src/Microsoft.ML.TensorFlow/TensorFlowModelInfo.cs

```diff
@@ -55,5 +56,10 @@ public Schema GetInputSchema()
         {
             return TensorFlowUtils.GetModelSchema(_env, Session.Graph, "Placeholder");
         }
+
+        public void Dispose()
+        {
+            Session.Dispose();
+        }
```

Comment:
Hi @ivanidzo4ka, I think you should probably follow the [advice of the pattern](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose). What you think?

Also, this is an extensible class (not sealed), so it should follow advice for extensible classes, which complicates the story somewhat. (Unless it not being sealed was a simple mistake, in which case that should be fixed.)